src/simulator_core/entities/heat_network.py

```diff
@@ -35,7 +36,7 @@ def __init__(self, conversion_factory: Callable[[pandapipesNet], Tuple[List[Asse
         self.panda_pipes_net = create_empty_network(fluid="water")
         self.assets, self.junctions = conversion_factory(self.panda_pipes_net)
 
-    def run_time_step(self, time: float, controller_input: dict) -> None:
+    def run_time_step(self, time: datetime.datetime, controller_input: dict) -> None:
```

Comment:
Ik snap niet helemaal waarom je hier niet gewoon een tijdstap zou willen. Heeft dat met de profielen te maken? Datetime vind ik (persoonlijk) geen heel fijn formaat.

Reply:
Mee eens, maar profile staat in datetime, dus die wil je wel gebruiken. In toekomst willen we misschien wel voor tijdstap nummer gaan, maar nu kan dat nog niet

Reply:
https://github.com/Project-OMOTES/simulator-core/issues/9